src/components/SearchPage.vue

```diff
@@ -38,7 +38,8 @@
         <b-navbar-nav v-if="isLoggedIn()" class="pl-4">
           <b-form-checkbox
             v-b-tooltip.hover.topleft.html.o100.d250 v-bind:title="$t('select_all')"
-            v-bind:indeterminate="isIndeterminate()"
+            v-bind:indeterminate="selectedOnPage() > 0 && selectedOnPage() < searchResults.length"
+            v-bind:checked="selectedOnPage() === this.searchResults.length"
```

Comment:
Weird comparisons with `searchResults.length`

Reply:
an easy solution could be having the array of uids and check if the current result uid has a match